src/Profiles.ts

```diff
@@ -520,6 +520,15 @@ export class Profiles {
             try {
                 const updSession = await zowe.ZosmfSession.createBasicZosmfSession(loadSession as IProfile);
                 if (rePrompt) {
+                    const button1 = localize("promptCredentials.saveCredentials.button", "Save Credentials for Future Use");
+                    const button2 = localize("promptCredentials.doNotSave.button", "Do Not Save");
```

Comment:
Is it okay to change the variable name button 1 and 2 to a more meaningful name?

Reply:
addressed in latest commit